nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/ConsumeJMS.java

```diff
@@ -86,6 +86,29 @@
             .defaultValue(CLIENT_ACK.getValue())
             .build();
 
+    static final PropertyDescriptor DURABLE_SUBSCRIBER = new PropertyDescriptor.Builder()
+                                                             .name("Durable subscription")
+                                                             .description("If destination is Topic if present then make it the consumer durable.")
```

Comment:
Similar to the "Shared subscription", can we make it clearer to describe what durable subscription is? For example:

> Specify whether to make the subscription durable, only effective If destination is a Topic. If enabled, the broker will hold messages for a consumer while it stops, so that it can consume those when it resumes.

Is my understanding correct??